source/admin_manual/pigeonhole_managesieve_server.rst

```diff
@@ -23,8 +23,8 @@ filesystem:
 Configuration and Use
 =====================
 
-* `Download and Installation
-  <https://wiki.dovecot.org/Pigeonhole/Installation>`_
+* :ref:`Download and Installation
+  <sieve_installation>`
 * `Configuration
   <https://wiki.dovecot.org/Pigeonhole/ManageSieve/Configuration>`_
 * `Troubleshooting
```

Comment:
ManageSieve still lives mostly in the wiki

Reply:
according to this page anyway

Reply:
That is true.